Libraries_API.ipynb

```diff
@@ -713,6 +715,34 @@
     "    -X DELETE"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### /query/&lt;library_id&gt;\n",
+    "Interact with a specific library. The `library_id` (string) is the unique identifier of the library, as specified in the `/libraries` GET response.\n",
```

Comment:
"Interact with a specific library" is vague. Maybe something like "Add or delete a documents to a given library based on a supplied query" or something like that.